test/unit/specs/store/getters.spec.js

```diff
@@ -244,4 +262,131 @@ describe(`Store: getters`, () => {
 
     expect(result).toEqual(context)
   })
+
+  // Jest doesn't mock Joi correctly and the test run fails.
```

Comment:
Do we use Joi in the end?

Reply:
No. I was unable to get Jest to not mock Joi. I'll remove this code.

See here for related info: https://github.com/facebook/jest/issues/90#issuecomment-67863810